examples/07_advanced/plot_mask_large_fmri.py

```diff
@@ -320,27 +289,27 @@ def numpy_masker_shared_parallel(img, masks):
     [
         "NiftiMasker,\nwith path",
         "NiftiMasker,\nwith in-memory\nimage",
-        "Numpy indexing,\nwith path",
-        "Numpy indexing,\nwith in-memory\nimage",
+        "Numpy indexing",
         "Numpy indexing,\nwith shared\nmemory",
```

Comment:
Light suggestion to be clearer that we're referring to the class `SharedMemory` rather than the idea of shared memory
```suggestion
        "Numpy indexing,\nwith\nSharedMemory",
```